### PR TITLE
KS-SK1156-KFT-Update-outgoing-transfert-Domains-TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations are done except EN subs.

Need proof for DE, EN-GB, Latin and PL versions before merge. 

Some translations seems to be made before, It was verified and all subs are now synchronised with the latest FRFR version.

### CODE REVIEW

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
{{.cds.build.pelicanoutput}}
```

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
{{.cds.build.pelicanoutput}}
```

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
{{.cds.build.pelicanoutput}}
```

**Comment by Kazarus4** [comment]
build ok, need proof

File: pages/web/domains/transfer_outgoing_domain/guide.es-es.md [hunk 1]
```diff
@@ -65,21 +63,32 @@ En la pestaña `Información general`{.action}, encontrará el cursor `Protecci
 
 Haga clic en el cursor y confirme en la nueva ventana que desea eliminar la protección. Espere unos minutos a que el estado pase a `Desactivado`{.action}.
 
+> [!primary]
+>
+> Si encuentra el mensaje "**Se ha producido un error al solicitar la desactivación de la protección del dominio ("User not granted for this request")**", significa que no tiene los permisos suficientes para desbloquear el dominio. 
+>
+> Además, si encuentra el mensaje: "**AUTH/INFO code : Authcode is not managed by OVHcloud, contact the registry to claim it**", que significa que el código de transferencia de su dominio no es recuperable a través de su [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es).  
+> 
+> En ambos casos, compruebe que es el contacto **administrador** del dominio con ayuda de nuestra guía sobre la [gestión de contactos](/pages/account/customer/managing_contacts) y compruebe que la extensión de su dominio permite un desbloqueo desde el[área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es).
```
Comment (OVH-Team-Guides) [inline pages/web/domains/transfer_outgoing_domain/guide.es-es.md]:
```suggestion
> En ambos casos, compruebe que es el contacto **administrador** del dominio con ayuda de nuestra guía sobre la [gestión de contactos](/pages/account/customer/managing_contacts) y compruebe que la extensión de su dominio permite un desbloqueo desde el [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es).
```

File: pages/web/domains/transfer_outgoing_domain/guide.es-us.md [hunk 2]
```diff
@@ -61,21 +65,32 @@ En la pestaña `Información general`{.action}, encontrará el cursor `Protecci
 
 Haga clic en el cursor y confirme en la nueva ventana que desea eliminar la protección. Espere unos minutos a que el estado pase a `Desactivado`{.action}.
 
+> [!primary]
+>
+> Si encuentra el mensaje "**Se ha producido un error al solicitar la desactivación de la protección del dominio ("User not granted for this request")**", significa que no tiene los permisos suficientes para desbloquear el dominio. 
+>
+> Además, si encuentra el mensaje: "**AUTH/INFO code : Authcode is not managed by OVHcloud, contact the registry to claim it**", que significa que el código de transferencia de su dominio no es recuperable a través de su [área de cliente de OVHcloud](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/world/&ovhSubsidiary=ws).  
+> 
+> En ambos casos, compruebe que es el contacto **administrador** del dominio con ayuda de nuestra guía sobre la [gestión de contactos](/pages/account/customer/managing_contacts) y compruebe que la extensión de su dominio permite un desbloqueo desde el[área de cliente de OVHcloud](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/world/&ovhSubsidiary=ws).
```
Comment (OVH-Team-Guides) [inline pages/web/domains/transfer_outgoing_domain/guide.es-us.md]:
```suggestion
> En ambos casos, compruebe que es el contacto **administrador** del dominio con ayuda de nuestra guía sobre la [gestión de contactos](/pages/account/customer/managing_contacts) y compruebe que la extensión de su dominio permite un desbloqueo desde el [área de cliente de OVHcloud](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/world/&ovhSubsidiary=ws).
```

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**